pages/index.js

```diff
@@ -78,56 +108,58 @@ export default function Home({ssGuildInfo}) {
   }, [guildInfo])
 
   return (
-    <div>
+    <Container>
       <Head>
         <title>Testausserveri</title>
       </Head>
       <HeroDiscordLive focused={heroFocused} />
       <Center>
         <Title style={{overflow: "hidden"}}>
           <TitleStaticGradientText>
-            Yhteisö
-            nuorille<br />
+            Aktiivinen yhteisö 
+            kaikille<br />
           </TitleStaticGradientText>
           <TextLoop>
-              <GradientText>hakkereille</GradientText>
-              <GradientText>koodareille</GradientText>
-              <GradientText>Linux-velhoille</GradientText>
-              <GradientText>radioamatööreille</GradientText>
-              <GradientText>graafikoille</GradientText>
-              <GradientText>3D-artisteille</GradientText>
+          <TitleGradientText>hakkereille</TitleGradientText>
```

Comment:
Could you elaborate why this change is necessary? Does it provide some kind of an visual improvement or style matter?

GradientText -> TitleGradientText